### ISSUE TITLE
MS Learn Module Update Request: How to create and modify a Git project

### ISSUE THREAD

**Issue by mjbear** [githubpartners/microsoft-learn#170]
### Which of the MS Learn modules from the dropdown are you submitting an update request?

None

### Additional information

- [ ] Fix a broken user experience (broken links, exercise error, etc.)
- [X] Update incorrect information
- [ ] Add new content to the module
- [ ] Some other request

### Information about the requested update

module: How to create and modify a Git project
lesson: Exercise - Start a project
https://learn.microsoft.com/en-us/training/modules/create-git-project/2-exercise-start-project

section: Create and add (stage) a file
touch updates a file's last-modified time if the file exists. If the file doesn't exist, ~~Git~~ [the touch command] creates an empty file with that file name.

(without the square brackets above of course)
It is "touch" that creates the empty file, not git.

**Comment by rmallorybpc** [comment]
Hi @mjbear ,

Thank you for reporting the issue. We are closing this issue due to the length of time open. Please let us know if the issue still exists by reporting an updated issue.

**Comment by mjbear** [comment]
> Hi [@mjbear](https://github.com/mjbear) ,
> 
> Thank you for reporting the issue. We are closing this issue due to the length of time open. Please let us know if the issue still exists by reporting an updated issue.

Hi @rmallorybpc,
There haven't been any requests for further detail once I submitted the ticket, so the lack of response wasn't waiting on the submitter. This is 1 of 21 with the same response.

Is there something I should have done to get attention to them sooner?

Or something I should have done differently?

If I came up with the time I could re-check the courses, what likelihood is there the next round of suggestions will see action within days, weeks, or months?

I appreciate the MS Learn content so that's why I submitted the suggestions. I don't doubt your group is busy so all I probably did with the suggestions is flood them with messages. :disappointed:

Respect for what you all maintain. Thank you! :thumbsup:

---

Relates to:
#153, #154, #155, #156, #157,
#158, #159, #160, #161, #162,
#163, #164, #165, #166, #167,
#168, #169, #171, #172, #174

**Comment by rmallorybpc** [comment]
@mjbear , yes, this was an exercise from the learn team to clean up old issues since most if not all of the content has been updated since posts from 2024 and before. the team is actively looking at new tickets and replying to each ticket with updates when the content needs updates. your new reviews of new content will receive the same reviews and updates. I would recommend just moving forward with the new courses rather than looking back at previous courses that have likely been updated or the content moved into our courses as structures have changed.

**Comment by mjbear** [comment]
> [@mjbear](https://github.com/mjbear) , yes, this was an exercise from the learn team to clean up old issues since most if not all of the content has been updated since posts from 2024 and before. the team is actively looking at new tickets and replying to each ticket with updates when the content needs updates. your new reviews of new content will receive the same reviews and updates. I would recommend just moving forward with the new courses rather than looking back at previous courses that have likely been updated or the content moved into our courses as structures have changed.

Ah, gotcha, makes sense.
I appreciate the response @rmallorybpc!
Have a good one!
